.github/workflows/vm-pr.yml

```diff
@@ -53,15 +53,15 @@ jobs:
       matrix:
         fork: [
           'Berlin',
-          'MuirGlacier',
+          # 'MuirGlacier',
```

Comment:
I'd say we also include `MuirGlacier` since this is what currently runs on mainnet.

Reply:
Thanks! the changes are in place.